src/applications/vaos/appointment-list/components/AppointmentsPage/RequestExpressCare.jsx

```diff
@@ -1,25 +1,18 @@
 import React from 'react';
-import environment from 'platform/utilities/environment';
 import { Link } from 'react-router-dom';
 import { FETCH_STATUS } from '../../../utils/constants';
 
 export default function RequestExpressCare({
   windowsStatus,
-  enabled,
   hasWindow,
   allowRequests,
   localWindowString,
   localNextAvailableString,
-  useNewFlow,
```

Comment:
This is the call to action, it's only displayed if we're using the new flow (i.e. va_online_scheduling_express_care_new is on)